lib/logproto/logproto-proxied-text-server.c

```diff
@@ -214,6 +249,12 @@ _log_proto_proxied_text_server_handshake(LogProtoServer *s)
   if (parsable)
     {
       msg_info("PROXY protocol header parsed successfully");
```

Comment:
We could not do this properly, because LogProto instances currently have no LogPipe/Driver/location information.

`evt_tag_printf("driver", "%p", self)` has no real meaning, it's actually misleading, because we print the LogProto address instead of the driver.

Reply:
At the very least this should not be on info level.